ale/config.yml

```diff
@@ -1,8 +1,8 @@
-spice_root: "/data/spice/"
+spice_root: '/data/spice/'
 cassini: '/usgs/cpkgs/isis3/data/cassini/kernels/mk/' # Cassini ISS
 mdis: '/scratch/jlaura/spice/mess-e_v_h-spice-6-v1.0/messsp_1000/extras/mk' # Messenger
 mex: '/scratch/jlaura/spice/mex-e_m-spice-6-v1.0/mexsp_1000/EXTRAS/MK' # Mars Explorer
 mro: '/data/spice/mro-m-spice-6-v1.0/mrosp_1000/extras/mk' # Mars Reconnaissance Orbiter
 kaguya: '/data/spice/SELENE/kernels/mk/'
-dawn: '/data/spice/dawn-m_a-spice-6-v1.0/dawnsp_1000/extras/mk'
+dawn: '/scratch/spice/dawn-m_a-spice-6-v1.0/dawnsp_1000/extras/mk/'
```

Comment:
does the dawn path need to be /scratch/jlaura/spice?

Reply:
Nope, I copied the metakernels into the /scratch/spice area and we will just need to update the path for them to point to /scratch/jlaura/spice for the actual data